scipy/stats/kde.py

```diff
@@ -492,15 +494,26 @@ def _compute_covariance(self):
         covariance_factor().
         """
         self.factor = self.covariance_factor()
-        # Cache covariance and inverse covariance of the data
-        if not hasattr(self, '_data_inv_cov'):
+        # Cache covariance and cholesky decomp of covariance of the data
+        if not hasattr(self, '_data_cov_chol'):
             self._data_covariance = atleast_2d(np.cov(self.dataset, rowvar=1,
                                                bias=False))
-            self._data_inv_cov = linalg.inv(self._data_covariance)
```

Comment:
@swallan this is still there, though.

Reply:
@swallan do you remember where this left off? I seem to remember you working on it.

Reply:
I don't think I did any work on this other than brief reviewing.

Reply:
OK, never mind.